### PR TITLE
fix: don't parse markdown inside html tags

### PR DESCRIPTION
**Pull request by gka**
Added one more alternative to the tokenizer regex that (sort of) catches HTML tags but doesn't match a group, so they're prevented from getting parsed. 

In contrast to the [fix by bpmn-io](https://github.com/developit/snarkdown/pull/99/files) we're expecting the character immediately following the `<` to be a letter so we're catching `<b>` but not `a < b`. 

```
<[a-zA-Z][^>]*>
```

Wrote some tests to confirm it's doing what it should.

Should note that there's still one ugly edge case that we're not catching. In this perfectly valid HTML string, snarkdown will still parse the `**foo**` because it finds the closing `>` first.

```html
<div title="5 > 10 **foo**">works</div>
``` 

I added a skipped test for this to remind us about this edge case.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -184,4 +184,36 @@ describe('snarkdown()', () => {
...
+			expect(snarkdown('5 is < **than** 10')).to.equal('5 is < <strong>than</strong> 10');
+			expect(snarkdown('5 is < **than** 10, but > 2')).to.equal('5 is < <strong>than</strong> 10, but > 2');
+			expect(snarkdown('5 is **<** 10, but **>** 2')).to.equal('5 is <strong><</strong> 10, but <strong>></strong> 2');
+		});
+
+		it('should not parse markdown in unquoted attributes', () => {
+			expect(snarkdown('<a title=**foo**>')).to.equal('<a title=**foo**>');
+		});
+
+		it.skip('should ignore > inside quoted arguments', () => {
```
Comment (elanals) [inline test/index.js:212]: how about marking it as failing instead?
```suggestion
		it.failing('should ignore > inside quoted arguments', () => {
```

**Review by elanals — APPROVED** [review]
amazing, thank you!

File: test/index.js (hunk 1 above)
Comment (gka) [reply test/index.js:212]: err, there's no `it.failing` in Mocha

Comment (gka) [reply test/index.js:212]: will revert to skip
